src/r-tech1/network/network.h

```diff
@@ -120,8 +120,8 @@ bool noDelay(Socket s, bool b);
 /* Whether or not new sockets re-use ports */
 void reuseSockets(bool what);
 
-void listen(Socket s);
-Socket accept(Socket s);
+void listen(Socket s) throw();
```

Comment:
you can leave off the throw() entirely. throw() by itself means 'doesnt throw anything' I think

Reply:
oh... i get it. removing